lib/hanami/assets/configuration.rb

```diff
@@ -57,6 +57,10 @@ class Configuration
       # @api private
       HTTPS_PORT               = '443'.freeze
 
+      # @since 0.3.0-add-options-to-javascript-helper
+      # @api private
+      DEFAULT_SRI_ALGORITHM    = 'sha256'.freeze
```

Comment:
I've chosen SHA256 as the default algorithm, but I have seen examples of SHA384 and SHA512 being used with existing CDNs. I've implemented `#sri_algorithm` to return an array so that more specified through the configuration in future (not yet implemented).